en/1-prep/1-compiling.md

```diff
@@ -78,6 +78,9 @@ cd substrate-parachain-template
 # Checkout the proper commit
 git checkout 1e1e725
 
+# Set up the environment
+./scripts/init.sh
```

Comment:
https://github.com/substrate-developer-hub/substrate-parachain-template/blob/master/scripts/init.sh

This script just updates the rust toolchain. I would either assume that people already did this, or explicitly state to do this, not use a script blindly . 

Please open an issue if you indeed ran into issues around toolchain so we can write this out in he tutorial itself :pray: